content/docs/pipelines/overview/interfaces.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Introduction to the Pipelines Interfaces"
+description = "The ways you can interact with the Kubeflow Pipelines system"
+weight = 20
++++
+
+This page introduces the interfaces that you can use to build and run
+machine learning (ML) workflows with Kubeflow Pipelines.
+
+## User interface (UI)
+
+You can access the Kubeflow Pipelines UI by clicking **Pipeline Dashboard** on 
+the Kubeflow UI. The Kubeflow Pipelines UI looks like this:
+  <img src="/docs/images/pipelines-ui.png" 
+    alt="Pipelines UI"
+    class="mt-3 mb-3 border border-info rounded">
+
+From the Kubeflow Pipelines UI you can perform the following tasks:
+
+* Run one or more of the preloaded samples to try out pipelines quickly.
+* Upload a pipeline as a zipped YAML file. The pipeline can be one that you
+  have built or one that someone has shared with you.
+* Create an *experiment* to group one or more of your pipeline runs.
+  See the [definition of an
+  experiment](/docs/pipelines/overview/concepts/experiment/).
+* Create and start a *run* within the experiment. A run is a single execution
+  of a pipeline. See the [definition of a
+  run](/docs/pipelines/overview/concepts/run/).
+* Explore the configuration, graph, and output of your pipeline run.
+* Compare the results of one or more runs within an experiment.
```

Comment:
should we add one entry to call out scheduling functionality?

Reply:
Done.